controllers/device_controller.go

```diff
@@ -170,33 +154,32 @@ func (r *DeviceReconciler) reconcileDeleteDevice(ctx context.Context, d *devicev
 	return nil
 }
 
-func (r *DeviceReconciler) reconcileCreateDevice(ctx context.Context, d *devicev1alpha1.Device, log logr.Logger) error {
+func (r *DeviceReconciler) reconcileCreateDevice(ctx context.Context, d *devicev1alpha1.Device) error {
 	// get the actual name of the device on the Edge platform from the Label of the device
 	edgeDeviceName := util.GetEdgeDeviceName(d, EdgeXObjectName)
 	newDeviceStatus := d.Status.DeepCopy()
-	log.V(4).Info("Checking if device already exist on the edge platform", "device", d.GetName())
+	klog.V(4).Infof("Checking if device already exist on the edge platform: %s", d.GetName())
 	// Checking if device already exist on the edge platform
 	edgeDevice, err := r.deviceCli.Get(nil, edgeDeviceName, iotcli.GetOptions{})
 	if err == nil {
 		// a. If object exists, the status of the device on OpenYurt is updated
-		log.V(4).Info("Device already exists on edge platform")
+		klog.V(4).Info("Device already exists on edge platform")
```

Comment:
Adding device name into the log message may be better.

Reply:
The device name has been added.